src/Sylius/Bundle/ApiBundle/spec/Serializer/ProductVariantNormalizerSpec.php

```diff
@@ -32,9 +35,10 @@ function let(
         ProductVariantPricesCalculatorInterface $pricesCalculator,
         ChannelContextInterface $channelContext,
         AvailabilityCheckerInterface $availabilityChecker,
-        SectionProviderInterface $sectionProvider
+        SectionProviderInterface $sectionProvider,
+        IriConverterInterface $iriConverter
     ): void {
-        $this->beConstructedWith($pricesCalculator, $channelContext, $availabilityChecker, $sectionProvider);
+        $this->beConstructedWith($pricesCalculator, $channelContext, $availabilityChecker, $sectionProvider, $iriConverter);
```

Comment:
if I see correctly we are not injecting it in `src/Sylius/Bundle/ApiBundle/Serializer/ProductVariantNormalizer.php`